dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnector.java

```diff
@@ -305,6 +306,36 @@ protected String toCallDescription() {
     }
   }
 
+  private static class CatalogsTask extends AbstractHiveMetadataTask implements FunctionsFormat {
```

Comment:
Will this work using an import or static import instead of the `implements` clause?

The contents of the interface looked like it doesn't need to be implemented here. There could be other reasons, e.g. `instanceof` or `AutoService`, but if this is not the case I would suggest to use an import instead.

Reply:
It was a leftover from copying a different task. Thanks for catching that. Removed.